CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Various documentation of the CRD ([#319]).
```

Comment:
Please add something like `BREAKING: Made lots of attributes mandatory. While this change is technically breaking, your Spark job would have not worked before as well, as these attributes are needed`

Reply:
See e4e6773.